library/src/main/AndroidManifest.xml

```diff
@@ -12,7 +12,6 @@
   <application>
 
     <service android:name="com.novoda.downloadmanager.LiteDownloadService" />
-    <service android:name="com.novoda.downloadmanager.LiteDownloadMigrationService" />
```

Comment:
💥 one less service. Now it's the clients responsibility to extract and determine the threading mechanism necessary. We just provide an example through the `MainActivity` of the demo and the `MigrationJob`.